static/js/emoji.js

```diff
@@ -11,9 +11,9 @@ exports.emojis_by_unicode = {};
 "stop_button", "stopwatch", "straight_ruler", "strawberry", "stuck_out_tongue", "stuck_out_tongue_closed_eyes", "stuck_out_tongue_winking_eye", "studio_microphone", "sun_behind_cloud", "sun_behind_cloud_with_rain", "sun_behind_large_cloud", "sun_behind_small_cloud", "sun_with_face", "sunflower", "sunglasses", "sunny", "sunrise", "sunrise_over_mountains", "sunset", "surfer", "sushi", "suspension_railway", "sweat", "sweat_droplets", "sweat_drops", "sweat_smile", "sweet_potato", "swimmer", "symbols", "synagogue", "syringe", "t_shirt", "taco", "tada", "tanabata_tree", "tangerine", "taurus", "taxi", "tea", "teacup_without_handle", "tear_off_calendar", "telephone", "telephone_receiver", "telescope", "television", "ten_oclock", "ten_thirty", "tennis", "tent", "thermometer", "thinking_face", "thought_balloon", "three", "three_oclock", "three_thirty", "thumbs_down", "thumbs_up", "ticket", "tiger", "tiger2", "tiger_face", "timer_clock", "tired_face", "tm", "toilet", "tokyo_tower", "tomato", "tongue", "top", "top_arrow", "top_hat", "tophat", "tornado", "trackball", "tractor", "traffic_light", "train", "train2", "tram", "tram_car", "triangular_flag", "triangular_flag_on_post", "triangular_ruler", "trident", "trident_emblem", "triumph", "trolleybus", "trophy", "tropical_drink", "tropical_fish", "truck", "trumpet", "tshirt", "tulip", "turkey", "turtle", "tv", "twelve_oclock", "twelve_thirty", "twisted_rightwards_arrows", "two", "two_hearts", "two_hump_camel", "two_men_holding_hands", "two_oclock", "two_thirty", "two_women_holding_hands", "u5272", "u5408", "u55b6", "u6307", "u6708", "u6709", "u6e80", "u7121", "u7533", "u7981", "u7a7a", "uk", "umbrella", "umbrella_on_ground", "umbrella_with_rain_drops", "unamused", "unamused_face", "underage", "unicorn_face", "unlock", "up", "up_arrow", "up_button", "upside_down_face", "us", "v", "vertical_traffic_light", "vhs", "vibration_mode", "victory_hand", "video_camera", "video_game", "videocassette", "violin", "virgo", "volcano", "volleyball", "vs", "vulcan_salute", "walking", "waning_crescent_moon", "waning_gibbous_moon", "warning", "wastebasket", "watch", "water_buffalo", "water_closet", "water_wave", "watermelon", "wave", "waving_black_flag", "waving_hand", "waving_white_flag", "wavy_dash", "waxing_crescent_moon", "waxing_gibbous_moon", "wc", "weary", "weary_cat_face", "weary_face", "wedding", "weight_lifter", "whale", "whale2", "wheelchair", "white_check_mark", "white_circle", "white_flower", "white_large_square", "white_medium_small_square", "white_medium_square", "white_medium_star", "white_small_square", "white_square_button", "wind_chime", "wind_face", "wine_glass", "wink", "winking_face", "wolf", "wolf_face", "woman", "womans_boot", "womans_clothes", "womans_hat", "womans_sandal", "women_partying", "womens", "womens_room", "world_map", "worried", "worried_face", "wrapped_present", "wrench", "writing_hand", "x", "yellow_heart", "yen", "yen_banknote", "yin_yang", "yum", "zap", "zero", "zipper_mouth_face", "zzz"];
```

Comment:
sorry thats kind of polish English probably - one element per line its probably correct ;-)